ConnectorAutocadCivil/ConnectorAutocadCivil/UI/ConnectorBindingsAutocadCivil.Selection.cs

```diff
@@ -71,24 +71,22 @@ public override List<ISelectionFilter> GetSelectionFilters()
     };
   }
 
-  public override void SelectClientObjects(List<string> args, bool deselect = false)
+  public override void SelectClientObjects(List<string> objs, bool deselect = false)
   {
-    var editor = Application.DocumentManager.MdiActiveDocument.Editor;
-    var currentSelection = editor.SelectImplied().Value?.GetObjectIds()?.ToList() ?? new List<ObjectId>();
-    foreach (var arg in args)
+    if (objs is not null && objs.Count > 0)
     {
-      try
+      Editor editor = Application.DocumentManager.MdiActiveDocument.Editor;
+
+      var currentSelection = editor.SelectImplied().Value?.GetObjectIds()?.ToList() ?? new List<ObjectId>();
```

Comment:
Can SelectImplied() be null?

Reply:
None of these methods or properties including [`SelectImplied()` ](https://help.autodesk.com/view/OARX/2023/ENU/?guid=OARX-ManagedRefGuide-Autodesk_AutoCAD_EditorInput_Editor_SelectImplied) are documented to indicate they may return null or throw; I may have added these checks previously due to undocumented null exceptions but will make it consistent with `SelectImplied()`. `currentSelection` shouldn't be null since it's set to an empty list on null.